### ISSUE TITLE
AllToMp3 is getting wrong artist / title for the files from YouTube

### ISSUE THREAD

**Issue by jkulak** [AllToMP3/alltomp3-app#133]
* My operating system is: macOS 10.14.6
* My AllToMP3 version is: 0.3.11

When trying to download YouTube file from the example of alltomp3-cli https://www.youtube.com/watch?v=zp7NtW_hKJI (Coldplay) this is what I get.

![Screenshot 2019-09-14 at 13 26 25](https://user-images.githubusercontent.com/426746/64907498-8ea3b080-d6f3-11e9-9350-cc2f683c0d91.png)

"boxes of rose" is my playlist from YouTube, and most of the files are being downloaded and saved to the same file: `A S H O O K - X X X N I C H K E P T I 6 9 M E T H E M X X X 4 2 0.mp3`. I have no idea what's that name and where does it come from.

**Comment by DaSniper12** [comment]
Yeah something similar is happening for me too, I am trying to download this Quinn XCII playlist, and all the songs are just getting titled _ and it rewrites them over and over again, so i just end up with the last song.
